backend/api/v1/endpoints/research.py

```
@@ -0,0 +1,360 @@
+        db.add(new_paper)
+        db.commit()
+        db.refresh(new_paper)
+
+        return {"message": "Paper uploaded successfully", "paper_id": new_paper.id}
+
+    except Exception as e:
+        logging.error(f"Error uploading paper: {str(e)}")
+        raise HTTPException(status_code=500, detail="Internal Server Error")
+
+@router.get("/papers/")
+def get_papers(db: Session = Depends(get_db), current_user: User = Depends(get_current_user)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_papers`

(ANN201)